pollux/prex/src/main/scala/org/hyperledger/identus/pollux/prex/PresentationSubmission.scala

```diff
@@ -0,0 +1,31 @@
+package org.hyperledger.identus.pollux.prex
+
+import io.circe.*
```

Comment:
is it possible to use zio-json?

Reply:
This object will be shared for didcomm (circe) and openid (zio-json). We need both now :smiling_face_with_tear: until we do circe deep clean.